### PR TITLE
fix: misc fixes

### LINKED ISSUE #1: Add instructions for running editor
It would be awesome if we could add a note to the README.md that mentioned how to server the editor. For example, it could be as simple as `npm install -g http-server && http-server .`.

### PR DESCRIPTION
**Pull request by zacanger**
These changes are probably best viewed with `-w` on the CLI, or `?w=1` appended to the URL on GitHub (https://github.com/grame-cncm/faustide/pull/59/files?w=1).

* Adds recommendation to clone with `--depth 1` due to ~300MB repo size
* Upgrades Sass and Node-Sass to get rid of Python 2 dependency in node-gyp.
* Runs `npm audit fix` (closes #58).
* Adds npm package to serve locally (closes #1).
* Updates all other dependencies with patch or minor versions available (no major/breaking upgrades besides sass and node-sass).
* Adds `private` to package.json to avoid accidental publish to the registry
* Adds a `safeStorage` alternative to using `localStorage` to reduce the risk of errors in private mode, etc.
* Removes CRLF in all source files except the Faust examples.
* Removes package-lock.json from showing up in `git diff`.
* Formats a few particularly long lines.
* Removes unused LocalStorageManager.
* Adds npm scripts to lint Sass and TS
* Corrects some typos and removes zero-width whitespace in `doc/wiki.md`.

### CODE REVIEW

File: doc/wiki.md [hunk 1]
```diff
@@ -86,29 +86,29 @@ All options and displays related to DSP runtime as MIDI, audio inputs and quick
...
+However, Monaco Editor's syntax highlighting system `Monarch` is different from the VSCode one, a new syntax description file is written under `Monarch` structure. Then, in order to provide inline documentation on hover a Faust function name or keyword, `Faust2MD` parser is ported to TypeScript. `Faust2MD` is originally created to generate `Markdown` format documentation of Faust functions which follows the [Standard convention](https://github.com/grame-cncm/faustlibraries) inside a Faust `*.dsp` file. The retrieved Markdown documentation can be directly served by the Monaco Editor.
 
 #### Audio probes
 ##### Retrieving Data
 To implement all four modes of signal visualisations: data table, oscilloscope (stacked and interleaved by channels), spectroscope and spectrogram, precise sample values are needed. Two ways to get audio output samples are available in the environment. The first one is to use WebAudio's `AnalyserNode` with its integrated methods:
 
-    get​Byte​Frequency​Data()
-    get​Byte​Time​Domain​Data()
+    getByteFrequencyData()
```
Comment (zacanger) [inline doc/wiki.md:100]: This doesn't show up on the GitHub diff, but there were zero-width whitespace characters in here.

**Comment by sletz** [comment]
@zacanger Thanks for the PR ! 
@Fr0stbyteR  is this PR OK for you?

**Comment by zacanger** [comment]
Bump @Fr0stbyteR is this all right?

**Comment by sletz** [comment]
This https://github.com/zacanger/faustide/pull/1 maybe?

**Comment by zacanger** [comment]
> This [zacanger#1](https://github.com/zacanger/faustide/pull/1) maybe?

Whoops, thanks, for some reason I never got a notification for that!

**Comment by sletz** [comment]
Thanks, so @zacanger is the PR ready to be merged?

**Comment by Fr0stbyteR** [comment]
I think we can merge!

**Comment by sletz** [comment]
Merged @zacanger thanks !

**Comment by sletz** [comment]
@zacanger save seems broken now ! Open several DSP, reload the page, only the default one is restored...

**Pull request merged**